README.md

```diff
@@ -1,102 +1,114 @@
-Setting up for local development
-================================
+# Setting up for local development
 
-To work on hosted rapidpro:
+## 1. Create an ngrok account
 
+Use ngrok to set up a URL that fowards to your development server. You'll need
+a paid account so that you can set up a subdomain.
+
+1. Download the ngrok zip: https://ngrok.com/download
+
+2. Unzip the package, and move the `ngrok` executable to a convenient
+   location.
+
+3. Log into https://ngrok.com using the Google login with your Caktus account.
+   This will give you access to the "Caktus Group" organization, which is
+   a paid account.
+
+4. Configure ngrok to use the Caktus account.
+   Get the auth token here from https://dashboard.ngrok.com/auth
+
+    ```
+    ./ngrok authtoken <caktus-auth-token>
+    ```
+
+   NOTE: You probably have access to an individual account as well as the
+   Caktus organization account. Make sure that "Caktus Group" is selected
+   from the account dropdown. Otherwise, you'll see this error when you start
```

Comment:
Ah, I never noticed that.

Maybe we should add a step between 3 & 4:

Navigate to the [Ngrok dashboard](https://dashboard.ngrok.com/status). The second line should say `Caktus Group / ngrok`.  If it says `Your Name / ngrok`, change it to `Caktus Group / ngrok` before continuing, to make sure you're using the paid Caktus Group account instead of a free personal account.